crates/sui-json-rpc/src/api/extended.rs

```diff
@@ -26,6 +26,18 @@ pub trait ExtendedApi {
         descending_order: Option<bool>,
     ) -> RpcResult<EpochPage>;
 
+    /// Return a list of epoch metrics, which is a subset of epoch info
+    #[method(name = "getEpochMetrics")]
+    async fn get_epoch_metrics(
+        &self,
+        /// optional paging cursor
+        cursor: Option<BigInt<u64>>,
+        /// maximum number of items per page
+        limit: Option<usize>,
+        /// flag to return results in descending order
+        descending_order: Option<bool>,
```

Comment:
For my understanding, this order is by epoch number? i.e. descending means the latest epoch will be returned first?

Reply:
yes, that's right.